networklibrary/src/main/java/com/eis/communication/network/Commands/AddResource.java

```diff
@@ -0,0 +1,27 @@
+package com.eis.communication.network.Commands;
+
+import com.eis.communication.network.SMSNetDictionary;
+
+/**
+ * Command to add a resource
```

Comment:
Add a resource where? What? Why? Who is allowed to add (idk, stupid queston)? Be as specific as you can.